RELEASENOTES.md

```diff
@@ -25,6 +25,28 @@ Page Scripting tests are now supported as part of CI/CD. By specifying pageScrip
 - `PageScriptingTestResults` is a JUnit test results file with all results combined.
 - `PageScriptingTestResultDetails` are the detailed test results (including videos) when any of the page scripting tests have failures. If the page scripting tests succeed - the details are not published.
 
+### Support for Git submodule
+
+[Git submodule](https://git-scm.com/book/en/v2/Git-Tools-Submodules) is now supported as part of CI/CD on your project.
+
+If you're already established your project repos outside the AL Go template, you may use submodule instead of placing and commiting your code directly to the repo generated by this template.
```

Comment:
```suggestion
If you already have created your project repos outside the AL-Go template, you may use a git submodule instead of moving your code directly to the repo generated using this template.
```

Reply:
This suggestion is no longer applicable since the part is deleted as the outcome of below suggestion 
https://github.com/microsoft/AL-Go/pull/1287#discussion_r1825718824